### ISSUE TITLE
thanks your work

### ISSUE THREAD

**Issue by SoulProficiency** [zhudongwork/SAM_TensorRT#5]
thanks for your contribution.
is this repo still suitable for edgeSAM?

**Comment by zhudongwork** [comment]
Yes, you can refer to https://github.com/dinglufe/segment-anything-cpp-wrapper.
